src/content/en/2021/media.md

```diff
@@ -1,20 +1,900 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Media
-description: TODO
+description: Media chapter of the 2021 Web Almanac covering how images and videos are currently being encoded, embedded, styled, and delivered on the web
 authors: [eeeps, dougsillars]
-reviewers: []
-analysts: [eeeps, dougsillars]
+reviewers: [Navaneeth-akam, tpiros, akshay-ranganath, addyosmani]
+analysts: [eeeps, dougsillars, akshay-ranganath]
 editors: []
 translators: []
+eeeps_bio: Eric Portis is a Web Platform Advocate at <a hreflang="en" href="https://cloudinary.com/">Cloudinary</a>.
+dougsillars_bio: Doug Sillars is a freelance digital nomad working on the intersection of performance and media. He tweets [@dougsillars](https://twitter.com/dougsillars), and blogs regularly at <a hreflang="en" href="https://dougsillars.com">dougsillars.com</a>.
```

Comment:
@dougsillars I copied this profile from a previous year. Feel free to update it.